directory/sail/templates/sail/home.html

```diff
@@ -7,5 +7,56 @@
 
 {% block main %}
 <!-- Actual page goes here -->
```

Comment:
@ranka47 : I meant to change `Tab Size: 4` to `Spaces:4` by clicking on it and selecting `Convert indentation to spaces`